EdgeCraftRAG/Dockerfile.server

```diff
@@ -16,7 +15,8 @@ RUN wget -qO - https://repositories.intel.com/gpu/intel-graphics.key | \
 RUN echo "deb [arch=amd64,i386 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/gpu/ubuntu jammy client" | \
     tee /etc/apt/sources.list.d/intel-gpu-jammy.list
 RUN apt-get update && apt-get install -y \
-    intel-opencl-icd intel-level-zero-gpu level-zero intel-level-zero-gpu-raytracing \
+    intel-opencl-icd intel-level-zero-gpu \
+    intel-level-zero-gpu-raytracing \
     intel-media-va-driver-non-free libmfx1 libmfxgen1 libvpl2 \
     libegl-mesa0 libegl1-mesa libegl1-mesa-dev libgbm1 libgl1-mesa-dev libgl1-mesa-dri \
     libglapi-mesa libgles2-mesa-dev libglx-mesa0 libigdgmm12 libxatracker2 mesa-va-drivers \
```

Comment:
Are VA-API drivers both from Intel Media driver project **and** Mesa project really needed (along with Mesa VDPAU drivers)?   And Mesa libraries for _all_ the 3D APIs (OpenGL, GLES2, GL Xserver integration, Vulkan)?